aten/src/ATen/Declarations.cwrap

```diff
@@ -1532,24 +1454,11 @@
     - THTensor* self
 ]]
 [[
-  name: tan_
-  types:
-    - floating_point
-  backends:
-    - CPU
-    - CUDA
+  name: _tan
```

Comment:
Can we call all of them `_th_*`? It's unclear what's different in a `_tan()` call compared to `tan()`

Reply:
_tan conflicts with another definition, presumably coming from NN (there's a tanh layer). This is not the case for the other functions.

Reply:
_th_ usually indicates that it'd otherwise conflict with something from TH. For example tanh.